### ISSUE TITLE
Escape key conflict with allowClose: false and keyboardControl: true

### ISSUE THREAD

**Issue by mzmcgrat** [kamranahmedse/driver.js#109]
Setting the allowClose option to false prevents the overlay from closing for both clicking the background and pressing the escape key, even if the keyboardControl option is set to true.

```
var driver = new Driver({
	allowClose: false,  /* prevents keyboard close */
	keyboardControl: true  /* escape key ignored */
});
driver.defineSteps([
	{
		element: "#thing1",
		popover: {
			title: "Title 1",
			description: "Text 1",
			position: "right"
		}
	},
	{
		element: "#thing2",
		popover: {
			title: "Title 2",
			description: "Text 2",
			position: "right"
		}
	}
]);
driver.start();
```

**Comment by ChriStef** [comment]
Hello I think you should have keyboardControl: false!

**Comment by kamranahmedse** [comment]
Fixed in 0.6.1
